### PR TITLE
Added Home Assistant support for HeatIt TRM3

### PR DESCRIPTION
**Pull request by hwestman**
Tested set point, mode and current temperature.

Not sure if you want to add the device id to the sample code. It wasn't clear to me, so I thought i'd suggest it.

### CODE REVIEW

File: hass/devices.js [hunk 1]
```diff
@@ -214,6 +214,27 @@ module.exports = {
       }
     }
   ],
+  '411-515-3': [
+    // Heatit Thermostat TF 058 (ThermoFloor AS)
+    {
+      type: 'climate',
+      object_id: 'thermostat',
```
Comment (robertsLando) [inline hass/devices.js:221]: @hwestman Isn't it missing the `"setpoint_topic": { "Heat (Default)": "67-1-1", "Cool": "67-1-2" },`?

**Review by robertsLando — CHANGES_REQUESTED** [review]

File: hass/devices.js (hunk 1 above)
Comment (hwestman) [reply hass/devices.js:221]: "setpoint_topic: Key-Value object where keys are the modes of the Zwave thermostat and values are the matching setpoint value_id (use this if your thermostat has more than one setpoint)"

My conclusion was that it didn't have more than one setpoint, so i removed it. I'll look into it!

Comment (robertsLando) [reply hass/devices.js:221]: Sorry just noticed that you only have heat mode :) It's ok so

**Review by robertsLando — APPROVED** [review]
LGTM

**Comment by robertsLando** [comment]
@hwestman Keep a look also to: https://github.com/zwave-js/zwavejs2mqtt

**Pull request merged**